pydantic/env_settings.py

```diff
@@ -113,7 +113,7 @@ def customise_sources(
         ) -> Tuple[SettingsSourceCallable, ...]:
             return init_settings, env_settings, file_secret_settings
 
-    __config__: Config  # type: ignore
+    __config__: ClassVar[Config]  # type: ignore
```

Comment:
I just reviewed https://github.com/samuelcolvin/pydantic/pull/2721, which already fixes this (in fact it's `ClassVar[Type[Config]]`.

Reply:
Done, type annotation have been updated.